net/net.sh

```diff
@@ -42,6 +42,7 @@ Operate a configured testnet
  startnode    - Start an individual node (previously stopped with stopNode)
  stopnode     - Stop an individual node
  startclients - Start client nodes only
+ prepare      - Prepare software deployment. (Build/download software the software release)
```

Comment:
```suggestion
 prepare      - Prepare software deployment. (Build/download the software release)
```
? @t-nelson

Reply:
If software isn't eating the world, it's certainly eating my brain :sweat_smile:

Reply:
:) Cool, I actually wasn't totally sure my correction is what you intended...